packages/next/server/next-server.ts

```diff
@@ -927,10 +927,10 @@ export default class NextNodeServer extends BaseServer {
   protected isServeableUrl(untrustedFileUrl: string): boolean {
     // This method mimics what the version of `send` we use does:
     // 1. decodeURIComponent:
-    //    https://github.com/pillarjs/send/blob/0.17.1/index.js#L989
-    //    https://github.com/pillarjs/send/blob/0.17.1/index.js#L518-L522
+    //    https://github.com/pillarjs/send/asset/0.17.1/index.js#L989
```

Comment:
```suggestion
    //    https://github.com/pillarjs/send/blob/0.17.1/index.js#L989
```

and the following lines